mobile/src/events/eventsSlice.ts

```diff
@@ -40,5 +44,5 @@ const eventSlice = createSlice({
   },
 });
 
-export const { setEvents, deleteEvent } = eventSlice.actions;
+export const { setEvents, changeEvent, deleteEvent } = eventSlice.actions;
```

Comment:
is 'changeEvent' used for modify event? If so, it might be helpful to change the name for consistency and transparency in the code